plenoptic/synthesize/eigendistortion.py

```diff
@@ -182,13 +182,13 @@ def synthesize(self,
         eigenvalues: Tensor
             Eigenvalues corresponding to each eigendistortion, listed in decreasing order. This tensor points to the
             `synthesized_eigenvalue` attribute of the object.
-        eigen_index: Listlike
+        eigen_index: Tensor
             Index of each eigendistortion/eigenvalue. This points to the `synthesized_eigenindex` attribute of the
             object.
         """
 
-        seed = np.random.randint(0, 2**32-1) if seed is None else seed
-        torch.manual_seed(seed)
+        if isinstance(seed, int):
```

Comment:
we should document that None is a possibility for seed (and maybe raise an exception if it's not seed or None, to catch weirdness like if they pass a float? or maybe try and cast it to an int

Reply:
done.